jekyll/_cci2/remote-docker.md

```diff
@@ -64,30 +64,28 @@ Let’s break down what’s happening during this build’s execution:
 - All docker-related commands are executed in your main container, too, but building/pushing images and running containers happens in the remote Docker Engine.
 - We use project environment variables to store credentials for Docker Hub.
 
-## Separation of environments
+## Separation of Environments
+Since the [job space][job-space] and [remote docker]({{ site.baseurl }}/2.0/glossary#remote-docker) are separated environments, there's one caveat: containers running in your job space can’t directly communicate with containers running in remote docker.
 
-Since [job space]({{ site.baseurl }}/2.0/glossary#job-space) and [remote docker]({{ site.baseurl }}/2.0/glossary#remote-docker) are two separated environments there's one caveat. Containers, running in your job space, cannot directly communicate with containers, running in remote docker. A few practical examples and ways work around:
+### Examples and Workarounds
+It’s impossible to start a service in remote docker and ping it directly from a main container (and vice versa). To solve that, you’ll need to interact with a a service from remote docker, as well as through the same container:
 
- * It's not possible to start service in remote docker and ping it directly from a main container and vice versa. To solve it you need to interact with a service from remote docker as well through the same container:
-
-   ``` YAML
-   # Starting our service and validating it's running
-   - run: |
-       docker run -d --name my-app my-app
-       docker exec my-app curl --retry 10 http://localhost:8080
-   ```
-
-   Another way of doing this is to use another container running in the same network with the target one:
+```yaml
+# start service and check that it’s running
+- run: |
+    docker run -d --name my-app my-app
+    docker exec my-app curl --retry 10 http://localhost:8080
+```
 
-   ``` YAML
-   - run: |
-       docker run -d --name my-app my-app
-       docker run --network container:my-app appropriate/curl --retry 10 http://localhost:8080
-   ```
+A different way to do this is to use another container running in the same network as the target container:
 
- * It's not possible to mount folder from build container into container in isolated docker and vise versa.
+```yaml
+- run: |
+    docker run -d --name my-app my-app
+    docker run --network container:my-app appropriate/curl --retry 10 http://localhost:8080
+```
 
-   TBD
```

Comment:
I think it'll be better to keep this `TBD` here, otherwise we just state the problem and not providing any solution. I think TBD in a beta doc is fine (of course we need a jira ticket for this)